gst/droidcamsrc/gstdroidcamsrc.c

```diff
@@ -1050,6 +1050,36 @@ gst_droidcamsrc_class_init (GstDroidCamSrcClass * klass)
           "Target bitrate", 0, G_MAXINT,
           DEFAULT_TARGET_BITRATE, G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
 
+  g_object_class_install_property (gobject_class, PROP_SUPPORTED_EXPOSURE_MODES,
+      g_param_spec_variant ("supported-exposure-modes",
```

Comment:
These string should also be consistent with the prop names. This will knock on slightly to the qtmultimedia PR sorry :/